modules/prelude-common-lisp.el

```diff
@@ -78,8 +78,8 @@
            slime-enable-evaluate-in-emacs t
            slime-autodoc-use-multiline-p t
            slime-auto-start 'always)
-
-     (define-key slime-mode-map (kbd "TAB") 'slime-indent-and-complete-symbol)
+     (set (make-local-variable 'tab-always-indent)
```

Comment:
Isn't this a global config somewhere? Probably here it can be omitted completely.